week-2-practise/level-2/2-validation.js

```diff
@@ -4,7 +4,17 @@
 // - is less than or equal to 100
 // Tip: write other small functions for each requirement
 
-function validate(num) {}
+function validate(num) {
+  if(typeof(num) == 'number'){
+    return positive(num);
+  }else{
+    return false;
+  }
+}
+
+function positive(num){
+  return 101 > num && num > 0;
```

Comment:
Think about the name of this method. Is your code doing what is expected? If the function is called "positive", I'd expect it to only check if the number is positive.